log/log.go

```diff
@@ -93,8 +127,9 @@ func WithPrefix(logger Logger, keyvals ...interface{}) Logger {
 //       returning a newly constructed context with a merged keyvals rather
 //       than simply wrapping the existing context.
 type context struct {
-	logger    Logger
-	keyvals   []interface{}
+	logger         Logger
+	keyvals        []interface{}
+	suffixKeyvals  []interface{}
```

Comment:
Might make it clearer and more consistent if we change `keyvals` to `prefixKeyvals`.

Reply:
i had that thought too, but wanted to keep changes to a minimum to make it easier to get merged upstream. but now that you mention, i'll do it. thank you for taking a look!

Reply:
actually, i'm going to skip renaming, because it doesn't read well in the context of the `With` function which doesn't have to do anything with "prefixing".